### ISSUE TITLE
[types] serializers

### ISSUE THREAD

**Issue by sayem314** [fastify/fastify#3026]
## 🐛 Bug Report

serializers is missing type `ip`

## To Reproduce

```ts
import fastify from 'fastify';

// Fastify instance
const app = fastify({
  logger: {
    prettyPrint: {
      levelFirst: true,
      translateTime: 'h:MM:ss TT',
      errorProps: 'message',
      ignore: 'pid,hostname,req,reqId,err,res,responseTime',
      messageFormat: '{req.method} {res.statusCode} {req.remoteAddress} {req.url} {msg}',
    },
    serializers: {
      req(req) {
        /* c8 ignore next 5 */
        return {
          method: req.method,
          url: req.url,
          remoteAddress: req.headers['X-Forwarded-For'] || req.ip, // <-- here
        };
      },
    },
  },
});

// Root endpoint for ping
app.get('/', {logLevel: 'error'}, async () => ({hello: 'world'}));

export default app;
```

## Expected behavior

Should have valid IP type.

## Your Environment

- *node version*: 14
- *fastify version*: 3.15.0
- *os*: Linux

**Comment by RafaelGSS** [comment]
Good catch! Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by nme077** [comment]
I will take a look at the issue and create a PR.

**Comment by RafaelGSS** [comment]
Are you still having this error? Looks like it was solved by #3034. And released in https://github.com/fastify/fastify/releases/tag/v3.16.0

**Comment by kibertoad** [comment]
@sayem314 Ping. Could you please check latest version?
